prisma/schema.prisma

```diff
@@ -28,8 +28,8 @@ model User {
 
 model Event {
   eventId     Int       @default(autoincrement()) @id
-  startDate   DateTime
-  endDate     DateTime
+  startDate   String
+  endDate     String
```

Comment:
why did you change it to a string? `DateTime` looks like the correct format for me